back/spec/services/track_segment_service_spec.rb

```diff
@@ -135,4 +168,23 @@
       service.track_activity(activity)
     end
   end
+
+  describe '#track_user' do
+    where(:user_factory, :is_tracked) do
+      [
+        [:user, false],
+        [:project_moderator, true],
+        [:project_folder_moderator, true],
+        [:admin, true],
+        [:super_admin, true]
+      ]
+    end
+
+    with_them do
+      it "returns #{params[:is_tracked]} for #{params[:user_factory].to_s.pluralize}" do
+        user = create(user_factory)
```

Comment:
I think we could use `build` instead of `create` here and save 5 queries

Reply:
Good point 🙂